CardinalKit-Example/CardinalKit/Study/StudyTableItem.swift

```diff
@@ -93,6 +129,43 @@ enum StudyTableItem: Int, CaseIterable {
             } else {
                 return Image(systemName: "dot.square")
             }
+        case .reaction:
+            if #available(iOS 14, *) {
+                return Image(systemName: "clock")
+            } else {
+                return Image(systemName: "clock.fill")
```

Comment:
I had to do #available check for some images because the one I prefer is not available in iOS 13. If both symbols are available since iOS 13, would be nice to stay consistent and just pick one.

Reply:
Then we can be consistent, originally I thought .fill symbols were only supported after iOS 13 :-)

Reply:
You can check this information with the SF Symbols app https://developer.apple.com/sf-symbols/